fjord/feedback/jinja2/feedback/android_feedback.html

```diff
@@ -0,0 +1,192 @@
+{% extends "base.html" %}
+
+{% block extra_headers %}
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
+{% endblock %}
+
+{% block site_js %}
+  {% javascript 'generic_feedback' %}
+{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'android-feedback' %}
+{% endblock %}
+
+{% block page_title %}{{ _('Firefox for Android Feedback') }}{% endblock %}
```

Comment:
We should add Jinja2 comment before this line explaining the l10n situation so that future developers understand what's going on. Namely that since we're copying strings over from Firefox for Android's codebase, we need to use the exact same strings.